src/components/IssueFieldAcknowledgments.vue

```diff
@@ -76,7 +76,7 @@ function handleDelete(uuid: string, closeModal: () => void) {
           </div>
           <button
             class="btn osim-cancel-new-acknowledgment"
-            @click="emit('acknowledgment:cancel-new', items[itemIndex])"
+            @click.prevent="emit('acknowledgment:cancel-new', items[itemIndex])"
```

Comment:
can also add `type="button"` here if this results in any weird side-effects

Reply:
Thanks for the review @superbuggy 
Indeed using type button also works properly. I changed all the prevents on click to use type button instead.